docs/README.md

```diff
@@ -66,7 +66,7 @@ You are now able to use all blocks that are exported by the Product Quantity app
 | `size` | `enum`| Preset size values for `font-size` and `padding`. You can check these value measures by accessing the [VTEX Styleguide](https://styleguide.vtex.com/#/Components/Forms/NumericStepper). Possible values are: `small`, `regular`, and `large`. | `small` |
 | `showLabel` | `boolean` | Whether a label should be displayed (`true`) or not (`false`). | `true` |
 | `selectorType` | `enum` | Defines how the quantity selector should initially behave. Possible values are: `stepper` (displays an input field where the quantity can be directly defined, in addition to side buttons to increase or decrease the value) and `dropdown` (shows a list of predefined-quantity options. In case the last quantity option is selected by users, the component is replaced with an input). | `stepper` |
-
+| `quantitySelectorStep` | `enum` | Show as unit the quantity of products, this is usefull in the case that the product have unitMultiplier. Possible values are `singleUnit` and `unitMultiplier`. On the `unitMultiplier`, the quantity will be affected with the unitMultiplier. On the `singleUnit`, the quantity will be not affected with the unitMultiplier.| `unitMultiplier`    |
```

Comment:
I don't think this description illustrates how this prop is working now. Could you also update it?

Reply:
Do you think that this ilustrate more? 
| `quantitySelectorStep` | `enum` | Show as single units the quantity of products that have unitMultiplier. Possible values are: `singleUnit` (the quantity will be affected with the unitMultiplier) and `unitMultiplier` (the quantity will be not affected with the unitMultiplier). |